### ISSUE TITLE
Poor performance with fp32 tensorRT segmentation model

### ISSUE THREAD

**Issue by poornimajd** [NVIDIA/TensorRT#1158]
## Description

I  have a pytorch segmentation model,which has been trained on a custom dataset.When I convert this model to the tensorrt version,I get a decrease in miou score from 89 to 66 ,though it is just fp32 version.I wonder why does the accuracy decrease so much.I used the following dependencies
Pytorch- 1.7.1
Cuda- 10.2
TensorRT-7.1.3
 My model has a bilinear interpolation layer ,but since it is a higher version of tensorRT ,I thought that this cannot bring down the accuracy so much.
What could be the problem here?
Any suggestion is greatly appreciated.
Thank you

## Environment

**TensorRT Version**: 7.1.3
**NVIDIA GPU**: 1060 gtx
**NVIDIA Driver Version**: 
**CUDA Version**: 10.2
**CUDNN Version**: 7.6.5
**Operating System**: Ubuntu 18.04
**Python Version (if applicable)**: 3.6.9
**Tensorflow Version (if applicable)**: NA
**PyTorch Version (if applicable)**: 1.7.1
**Baremetal or Container (if so, version)**: NA

## Steps To Reproduce
I used the following code for inference-
```
import torch
import torchvision
import torch2trt
import utils
from models.shelfnet_expt import ShelfNet
device="cuda"
import numpy as np
import cv2
import sys
import time

model = ShelfNet(n_classes=19)
model.load_state_dict(torch.load('./ShelfNet18_realtime.pth'), strict=False)
model=model.cuda().eval()
data = torch.ones((1, 3, 1024, 1024)).cuda()
model_trt = torch2trt.torch2trt(model, [data], fp16_mode=False)

device = torch.device('cuda')
mean = 255.0 * np.array([0.485, 0.456, 0.406])
stdev = 255.0 * np.array([0.229, 0.224, 0.225])

normalize = torchvision.transforms.Normalize(mean, stdev)

def preprocess(img):
    global device, normalize
    
    x = cv2.imread(img)
    x = cv2.cvtColor(x, cv2.COLOR_BGR2RGB)
    x = x.transpose((2, 0, 1))
    x = torch.from_numpy(x).float()
    x = normalize(x)
    x = x.to(device)
    x= x.contiguous()
    x = x[None, ...]
    return x

camera='./44_leftImg8bit.jpg'
for i in range(1):
    x=preprocess(camera)
    output = model_trt(x)[0].detach().cpu().float().numpy()
#save the output as image
palette = np.random.randint(0, 256, (256, 3), dtype=np.uint8)
output=np.array(output,dtype='int8')
pred = palette[output[0]]
cv2.imwrite("./44.png",pred)
```

**Comment by ttyio** [comment]
Hello @poornimajd , we have some fix for the  bilinear interpolation in 8.0, could you take a try? thanks
https://developer.nvidia.com/nvidia-tensorrt-download

**Comment by nvpohanh** [comment]
@poornimajd Have you tried the latest TensorRT and see if this has been fixed? Thanks

**Comment by nvpohanh** [comment]
I am going to close this due to >2wk of no action. Please feel free to reopen if this issue still exists. Thanks!
